ledfx/api/websocket.py

```
@@ -447,6 +468,120 @@ def song_info_handler(self, message):
+            WebsocketConnection.client_metadata[self.uid] = {
+                "ip": self.client_ip,
+                "device_id": self.device_id,
+                "name": self.client_name,
+                "type": self.client_type,
+                "connected_at": self.connected_at or time.time(),
+                "last_active": time.time(),
+            }
+
+    def _name_exists(self, name, exclude_uuid=None):
+        """Check if name is already taken"""
+        for uuid, meta in WebsocketConnection.client_metadata.items():
```

Flake8 (7.3.0)

[error] import 'uuid' from line 6 shadowed by loop variable

(F402)